research-app/src/App.vue

```diff
@@ -365,6 +405,15 @@ import {
   convertPywwtSpreadSheetLayerSetting,
   convertSpreadSheetLayerSetting,
 } from "./settings";
+import { imageSetLayerSettingNames, ImageSetLayerState } from "@wwtelescope/engine-helpers/src/imagesetlayer";
+import { isSpreadSheetLayerSetting, spreadSheetLayerSettingNames, SpreadSheetLayerState } from "@wwtelescope/engine-helpers/src/spreadsheetlayer";
+import { isLoadImageCollectionCompletedMessage, isLoadImageCollectionMessage, PywwtSpreadSheetLayerSetting } from "@wwtelescope/research-app-messages/dist/classic_pywwt";
+import { type } from "os";
```

Comment:
Is this `os.type` import used anywhere? It's hard to search for, but I couldn't see a usage? And it looks like this may be a special Node.js module — is it one of these things that TypeScript specially handles?

Reply:
Hmm, this isn't used anywhere (in fact, VS Code has it marked as unused). This feels like a mistaken auto-import to me (e.g. I typed, `os.type` instead of `o.type` for a message or something), and fixed the typo but not the import.